<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLoginDialog</class>
 <widget class="QWidget" name="DLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>362</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>362</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>362</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DLoginDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 63 13pt &quot;URW Gothic&quot;;
}

DLoginDialog
{
	border-image: url(:/images/profiles/loginBg.jpg);
}

QFrame
{
	background:#333;
	border-radius:15px;
}

QToolButton
{
	background-color: rgb(255, 0, 0);
	border-radius:35px;
}

QPushButton
{
	background:red;
	border-radius:5px;
}

QLabel
{
	color: rgb(238, 238, 236);
	font: 75 15pt &quot;aakar&quot;;
}

QLineEdit
{
	background:transparent;
	border:none;
	color:#717072;
	border-bottom:1px solid #717072;
}

QPushButton:hover:!pressed{
	background-color: rgb(204, 0, 0);
}

QPushButton:hover:pressed{
	background-color: rgb(78, 154, 6);
}
</string>
  </property>
  <widget class="QToolButton" name="mToolBtnProfile">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/imgs/PNG/users.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="mFrameCenter">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="mLabelLoginHere">
        <property name="text">
         <string>LOGIN HERE</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="mWidgetCenterCenter" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="mLabelPsd">
         <property name="font">
          <font>
           <family>aakar</family>
           <pointsize>15</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="mLineEditPsd">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="mLabelUsrName">
         <property name="font">
          <font>
           <family>aakar</family>
           <pointsize>15</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>username:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="mLineEditUsrName">
         <property name="placeholderText">
          <string>UserName</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnLogin">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnRegister">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
